packages/std/Cargo.toml

```diff
@@ -54,7 +54,7 @@ sha2 = "0.10.3"
 serde = { version = "1.0.103", default-features = false, features = ["derive", "alloc"] }
 serde-json-wasm = { version = "0.5.0" }
 thiserror = "1.0.26"
-uint = "0.9.3"
+bnum = "=0.7.0"
```

Comment:
Why not allow patch releases?

```suggestion
bnum = "0.7.0"
```

Reply:
I was afraid the padding problem of the Display impl (the one you commented on below) might get fixed in a patch release, resulting in a double padded implementation, but it's probably not such a big deal.

Reply:
We can trim trailing leading zeros to protect against that, right?

Reply:
Oh, we don't need to because `let unpadded = self.0.to_string();` is always `let unpadded = format!("{}", self.0)`, i.e. no extra padding here.

Reply:
Oh, true